src/main/java/fish/payara/extras/upgrade/UpgradeServerCommand.java

```diff
@@ -116,7 +116,7 @@ public class UpgradeServerCommand extends BaseUpgradeCommand {
     private File useDownloadedFile;
 
     private static final String NEXUS_URL = System.getProperty("fish.payara.upgrade.repo.url",
-            "https://nexus.payara.fish/repository/payara-enterprise-downloadable-artifacts/fish/payara/distributions/");
+            "https://nexus.dev.payara.fish/repository/payara-enterprise-downloadable-artifacts/fish/payara/distributions/");
```

Comment:
This should stay as-is since it's user-facing.

Reply:
reverted as its is